src/manager.js

```diff
@@ -51,6 +52,7 @@ export const getManagerAddress = async () => {
   try {
     await executeScript({
       code,
+      service: true
```

Comment:
What does the `service` flag do?

Reply:
We check whether `FlowManager` is deployed simply by trying to execute script, which specifying `import FlowManager from SERVICE_ACCOUNT_ADDRESS` and empty body [(Cadence Code Here)](https://github.com/onflow/flow-js-testing/blob/master/cadence/scripts/check-manager.cdc). If script would throw an error, we will deploy a contract and continue with tests.

So, since there is import statement in template code `executeScript` will try to resolve this contract address by calling `getContractAddress`, which basically would die in infinite loop 😅
This flag will tell it to skip the resolver.

Reply:
I see, thanks!